.circleci/config.yml

```diff
@@ -28,6 +28,9 @@ commands:
             rm -Rf emscripten
             echo "V8_ENGINE = '`pwd`/upstream/bin/d8'" >> ~/.emscripten
             echo "JS_ENGINES = [NODE_JS, V8_ENGINE]" >> ~/.emscripten
+            echo "WASM_ENGINES = []" >> ~/.emscripten
+            test -f ~/vms/wasmtime && echo "WASMTIME = os.path.expanduser(os.path.join('~', 'vms', 'wasmtime')) ; WASM_ENGINES.append(WASMTIME)" >> ~/.emscripten || true
```

Comment:
If `~/.wasmer/bin` is the default location why not just hardcode that here?

Same with `wasmtime-dev-x86_64-linux/wasmtime`.. why copy them around?

Reply:
I thought it might be nice to have a `vms/` directory. Just one place we need to care about, and also we can avoid persisting to the CircleCI image anything but the necessary files.